decidim-dev/lib/decidim/dev/test/rspec_support/warden.rb

```diff
@@ -1,7 +1,23 @@
 # frozen_string_literal: true
 
+module Decidim
+  module WardenTestHelpers
+    include Warden::Test::Helpers
+
+    #
+    # Utility method to login in the middle of a test as a different user from
+    # the current one.
+    #
+    def relogin_as(user, scope:)
```

Comment:
Oh, I got you now (it was crystal clear... xD). Yes, a default value of user makes sense! I didn't worry because all current calls passed it explicitly, but yes, makes sense.